terraform/github/analytical-platform-repositories.tf

```diff
@@ -452,6 +452,20 @@ locals {
         admins  = [module.data_platform_teams["data-platform-apps-and-tools"].id]
         pushers = [module.data_platform_team.id]
       }
+    },
+    "analytical-platform-image-build-template" = {
+      name                                   = "analytical-platform-image-build-template"
+      description                            = "Analytical Platform Image Build Template"
+      is_template                            = true
+      use_template                           = false
+      visibility                             = "public"
+      advanced_security_status               = "disabled"
```

Comment:
Should we be turning this on or not needed for a template?

Reply:
ive removed it as the docs says If a repository's visibility is public, advanced security is always enabled and cannot be changed